server/src/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -497,7 +498,7 @@ protected Long restart(final HaWorkVO work) {
 
                 boolean fenced = false;
                 if (alive == null) {
-                    s_logger.debug("Fencing off VM that we don't know the state of");
+                    s_logger.debug("Fencing off VM " + vm + " that we don't know the state of");
```

Comment:
Please wrap in a isDebugEnabled check to avoid expensive string concatenation when DEBUG is off.  Also, should this message be a WARN instead of a DEBUG?